projects/packages/backup/src/class-rest-controller.php

```diff
@@ -616,6 +628,66 @@ public static function get_site_backup_undo_event() {
 		return rest_ensure_response( $undo_event );
 	}
 
+	/**
+	 * Fetch a backup of a order, along with all of its data.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @param WP_REST_Request $request The request sent to the WP REST API.
+	 *
+	 * @return array
+	 */
+	public static function fetch_wc_orders_backup( $request ) {
+		global $wpdb;
+
+		// Disable Sync as this is a read-only operation and triggered by sync activity.
+		Sync_Actions::mark_sync_read_only();
+
+		$order_id = $request['id'];
+
+		$order                  = array();
+		$order_addresses        = array();
+		$order_operational_data = array();
+		$order_meta             = array();
+
+		if ( ! class_exists( 'Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore' ) ) {
+			return new WP_Error( 'order_not_allowed', __( 'Not allowed to get the order with current configuration', 'jetpack-backup-pkg' ), array( 'status' => 403 ) );
```

Comment:
If there is no class `OrdersTableDataStore` the WooCommerce is not installed or is an old version, so we can't get the HPOS orders

Reply:
Maybe we could use `OrdersTableDataStore::class` instead of using the string with the full namespace for improving readability, given we are already declaring the namespace.

Reply:
Good idea. Done! 👍